src/fast_check/mod.rs

```diff
@@ -26,58 +27,61 @@ fn format_diagnostics(diagnostics: &[FastCheckDiagnostic]) -> String {
 
 #[derive(Debug, Clone, thiserror::Error)]
 pub enum FastCheckDiagnostic {
-  #[error("Could not resolve '{name}' referenced from '{referrer}'. This may indicate a bug in Deno. Please open an issue to help us improve if so.")]
+  #[error("could not resolve '{name}' referenced from '{referrer}'")]
```

Comment:
I'm not really fan of lowercase error messages and most of our error messages don't do this (typescript diagnostics and v8 errors also don't do it). I understand `thiserror` recommends it and the rust std library does it, but I don't get why.

Reply:
these error messages are prefixed by `error[zap-not-found-reference]:` in output. I think not using caps helps the text flow better (looks like one sentence then).